tests/unit/test_test_utils.py

```diff
@@ -114,3 +115,9 @@ def test_check_synapses(create_tmp_simulation_config_file):
     # wrong type
     with pytest.raises(TypeError):
         utils.check_synapses([0], edges, selection)
+
+
+def test_check_signal_peaks():
```

Comment:
I am not sure that this function works even if the signal rises or descends. May you test also that case? An inclined trace with some peaks

Reply:
done